pkgs/applications/audio/pulseaudio-dlna/default.nix

```diff
@@ -30,7 +30,7 @@ pythonPackages.buildPythonApplication rec {
 
   propagatedBuildInputs = with pythonPackages; [
     dbus-python docopt requests setproctitle protobuf psutil futures
-    chardet notify2 netifaces pyroute2 pygobject2 lxml zeroconf ]
+    chardet notify2 netifaces pyroute2 pygobject2 lxml zeroconf_19 ]
```

Comment:
Never put multiple versions of the same package in `python-packages.nix`. Please use `packageOverrides` inside the pulseaudio-dlna expression instead.
Also add a comment on why this is needed, maybe referencing https://github.com/masmu/pulseaudio-dlna/issues/342.

Reply:
Thanks, I'll do as you said.

Reply:
Actually, we could also use separate expression for Python 2 and 3, like
```
zeroconf = if isPy3k then callPackage ... else callPackage ...;